File: 프로그래머스/0/120809. 배열 두배 만들기/배열 두배 만들기.java
class Solution {
    public int[] solution(int[] numbers) {
        int[] answer = numbers;
        for (int i = 0; i < numbers.length; i++) {
            answer[i] = numbers[i] * 2;
        }
        return answer;
    }
    
    // 다른 풀이
    // public int[] solution(int[] numbers) {
    //     return Arrays.stream(numbers).map(i -> i * 2).toArray();
    // }
}

File: 프로그래머스/0/120809. 배열 두배 만들기/배열 두배 만들기.js
function solution(numbers) {
    var answer = [];
    for(let i = 0; i < numbers.length; i++) {
        answer[i] = numbers[i] * 2;
    }
    return answer;
}

// 다른 풀이
// function solution(numbers) {
//     var answer = [];
//     return numbers.map(number => number * 2);
// }